src/markdown/mas/analyses_beheerovereenkomsten.Rmd

```diff
@@ -0,0 +1,4152 @@
+  fit_sbp_gelekwik_ZINB1 <- add_criterion(
+    sbp_gelekwik_ZINB1, c("kfold"), K = 5, cores = 5,
+    folds = "stratified", group = "regio", 
+    file = paste0(model_path, "fit_sbp_gelekwik_ZINB1"))
+} else {
+  fit_sbp_gelekwik_ZINB1 <- readRDS(paste0(model_path, 
+                                           "fit_sbp_gelekwik_ZINB1.rds"))
+}
+```
+
+```{r}
+models_dist <- data.frame(
+  model = c("Poisson", 
+            "Negative binomial",
+            "Zero-inflated Poisson",
+            "Zero-inflated Negative binomial"),
+  naam =  c("fit_sbp_gelekwik_pois1", 
+            "fit_sbp_gelekwik_negbin1",
+            "fit_sbp_gelekwik_ZIP1",
+            "fit_sbp_gelekwik_ZINB1")
+  )
+
+# Comparison among models
+comparisons <- loo_compare(fit_sbp_gelekwik_pois1, 
+                           fit_sbp_gelekwik_negbin1, 
+                           fit_sbp_gelekwik_ZIP1, 
+                           fit_sbp_gelekwik_ZINB1, 
+                           criterion = "kfold")[, c(1, 2)] %>%
+                as.data.frame() %>%
+                rownames_to_column("naam") %>% 
+                full_join(models_dist, by = join_by(naam))
+
+rbind(
+  fit_sbp_gelekwik_pois1$criteria$kfold$estimates[, "Estimate"],
+  fit_sbp_gelekwik_negbin1$criteria$kfold$estimates[, "Estimate"],
+  fit_sbp_gelekwik_ZIP1$criteria$kfold$estimates[, "Estimate"], 
+  fit_sbp_gelekwik_ZINB1$criteria$kfold$estimates[, "Estimate"]
+  ) %>%
+  as.data.frame() %>%
+  mutate(model = models_dist$model) %>%
+  full_join(comparisons, by = "model") %>%
+  mutate(CI_ll = elpd_diff  + qnorm(0.025) * se_diff,
+         CI_ul = elpd_diff  + qnorm(0.975) * se_diff) %>%
+  select(model, naam, everything()) %>%
+  arrange(desc(elpd_diff)) %>%
+  kable(digits = 3)
+```
+
+De fit van de modellen verschilt niet veel van elkaar.
+We besluiten dat het Poisson model toont het beste evenwicht tussen model parsimonie en model fit.
+De posterior predictive check toont een goede fit van dit meest eenvoudige model.
+We vergelijken de waargenomen tellingen met gesimuleerde tellingen van de posterior predictive distribution.
+
+```{r}
+pp_check(fit_sbp_gelekwik_pois1, type = "bars_grouped", ndraws = 100, 
+         group = "regio", facet_args = list(ncol = 1, scales = "free_y"))
+```
+
+```{r}
+pp_check(fit_sbp_gelekwik_pois1, type = "bars_grouped", ndraws = 100, 
+         group = "jaar.f", facet_args = list(ncol = 1, scales = "free_y"))
+```
+
+### Resultaten (1)
+
+We zien geen significant verschil in aantal broedvogels Gele kwikstaart met toenemende oppervlakte aan soortbeschermingsplannen.
+
+De Moeren:
+
+```{r}
+hypothesis(fit_sbp_gelekwik_pois1,
+           "area_prop_sb = 0")
+```
+
+Oostelijke leemstreek:
+
+```{r}
+hypothesis(fit_sbp_gelekwik_pois1,
+           "area_prop_sb + regioOostelijkeleemstreek:area_prop_sb = 0")
+```
+
+Visueel:
+
+```{r}
+plot(conditional_effects(fit_sbp_gelekwik_pois1, 
+                         effects = "area_prop_sb:regio"))
+```
+
+Cumulatief:
```

Comment:
Interessante analyse voor in een A1 paper lijkt me

Reply:
Ja er zit veel potentieel in. Ik denk dat we voor het vervolg specifieke hypotheses moeten opstellen en een modelstructuur bepalen die zoveel mogelijk data meeneemt (bv. telperiodes, variabelen op basis van remote sensing). Als we die dingen combineren zittten we goed denk ik. We zullen zien ...